File: overload.js
// function runMe(a,b,c,d,e) {
//     console.log(a,b,c,d,e);
//     console.log("runMe with 5 parameters: ", arguments);
// }

// function runMe() {
//     console.log("runMe with no parameters: ", arguments);
// }

class Run {


    runMe = (...args) => {
      args.forEach(a => console.log(`runMe(${args.length})`,a));
    }



}

const run = new Run();

run.runMe(1,2,3,4,5);

File: Overload.java
class Run {

    void runMe() {
        System.out.println("runMe()");
    }

    void runMe(int a, int b) {
        System.out.println("runMe(int a, int b)" + a + " " + b);
    }

    void runMe(int a, int b, int c) {
        System.out.println("runMe(a, b, c) " + a + " " + b + " " + c);
    }

    public static void main(String[] args) {
        Run run = new Run();
        int a = 1;
        int b = 2;
        int c = 3;
        run.runMe(a,b,c);
    }

}






